ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -14,7 +14,7 @@ CONSTANTS CLIENT_PRIMARY_KEY
 ASSUME CLIENT_PRIMARY_KEY \in [CLIENT -> KEY]
 
 \* $next_ts$ is the timestamp for transaction. It is increased monotonically,
-\* so every transaction must have a unique start and commit ts.
+\* so every transaction must have a unique start ts.
```

Comment:
.. so it is allowed that `txnA.start_ts = txnB.commit_ts` ?

Reply:
Yes, it could happen.